javageci-core/src/main/java/javax0/geci/accessor/AbstractAccessor.java

```diff
@@ -47,7 +47,7 @@ protected String defaultFilterExpression() {
 
     protected void writeGetter(Field field, String name, String getterName,
                                String type, String access, Segment segment) {
-        segment._r("%s %s %s(){", access, type, getterName)
+        segment._r("%s%s %s() {", access, type, getterName)
```

Comment:
Rename the variable `access` because this is not anymore the access modifier. This is either empty string or an access modifier and space.

A better solution would be to have a new method that returns and empty string when the argument is empty string and the argument string padded with a single space in all other cases. Coverting the access modifier is one thing, padding with space is another and these two tasks are coded in the single `check()` method. That is why it is not clean.